README.md

```diff
@@ -34,6 +34,7 @@ Dapr is currently under community development with preview releases.  The master
 | [5. Bindings](./5.bindings)            | Demonstrates how we use Dapr to create input and output bindings to other components. Uses bindings to Kafka.                                                                            |
 | [6. Functions-and-keda](./6.functions-and-keda) | Demonstrates use of Dapr pub/sub from Azure Functions, as well as composition with KEDA. |
 | [7. Middleware](./7.middleware) | Demonstrates use of Dapr middleware to enable OAuth 2.0 authorization. |
+| [8. Observability](./8.observability) | Demonstrates Dapr tracing capabilities. Uses Zipkin as a tracing component. |
```

Comment:
It would be really good to have this sample as 5 and have the others renumbered. But we can do this in a future PR, rather than this one.

Reply:
Agree